### DISCUSSION TITLE
ChangeLog Error

### DISCUSSION THREAD

**Discussion by fbaschin** [netbox-community/netbox#11615] [General]
When I click Others -> Changelog I get this error

<class 'AttributeError'>

'NoneType' object has no attribute '_base_manager'

Python version: 3.9.2
NetBox version: 3.4.1

**Comment by fbaschin** [comment]
Problem was only by one user. I deleted the user and recreate it. And all works fine again

**Reply by jeremystretch** [reply]
FYI this is bug #11335
